test/test_meta.py

```diff
@@ -7,7 +7,7 @@ def test_generated_workspace(managed_workspace, managed_user, server):
     """Test that a generated workspace exists when querying the API."""
 
     with conftest.login(managed_user, server):
-        resp = server.get(f"/api/workspaces/{managed_workspace}")
+        resp = server.get(f"/api/workspaces/{managed_workspace}/permissions")
```

Comment:
Shouldn't we check that the permissions are what we expect?

Perhaps we need a bit more complex fixtures to make this happen easily. I have some ideas.

It's ok to not address this now, but please file an issue instead.

Reply:
Oh I see, you ran that test below. n/m.

Reply:
Yeah, this just updates the endpoint with the new URL.